ui/src/pages/bed-splash.tsx

```diff
@@ -34,6 +35,13 @@ export const BedSplash = () => {
     full: true,
   });
 
+  const {
+    data: genomeStats,
+  } = useBedGenomeStats({
+    md5: bedId,
+    autoRun: true,
```

Comment:
If we always want to auto run this you can probably just set it in the query hook